index.html

```diff
@@ -110,54 +140,62 @@ <h3 class="project-name">Project name goes here</h3>
-    </div>
-    <div class="skills-section">
-      <h2>Skills</h2>
-      <ul class="skills-list">
-        <li>Codekit</li>
-        <li>Codepen</li>
-        <li>Github</li>
-        <li>Gitlab</li>
-        <li>Terminal</li>
-      </ul>
-      <img src="./images/ic_skills.png" alt="" srcset="">
-    </div>
-  </section>
+  <div class="all-about-img">
+    <img src="./images/Group116.png" class="about-images-desktop" alt="" srcset="">
+  </div>
+  <div class="all-about">
+    <section class="about-section">
+      <h1 class="section-title">About</h1>
+      <div class="greetings">
+        <h3>Hello! <u>Fickry Bil Iman</u> Here.. If you like what you see and have a project you need coded, don’t hesitate to contact me.</h3>
+        <h3>I am a software developer! I can help you build a product, feature or website.</h3>
+      </div>
+      <button type="button" class="button-about">Get my CV &nbsp;&nbsp;&nbsp; <img src="./images/ic_download.png" alt="" style="align-self: flex-end;"></button>
+    </section>
+    <section class="works-lfs-grid">
+      <div class="languages-section">
+        <h2>Languages</h2>
+        <ul class="languages-list">
+          <li>Javascript (ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+        <img src="./images/hashtag.png" alt="" srcset="">
+      </div>
+      <div class="frameworks-section">
+        <h2>Frameworks</h2>
+        <ul class="frameworks-list">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+          <li>Selenium</li>
+        </ul>
+        <img src="./images/Group 18.png" alt="" srcset="">
+      </div>
+      <div class="skills-section">
+        <h2>Skills</h2>
+        <ul class="skills-list">
+          <li>Codekit</li>
+          <li>Codepen</li>
+          <li>Github</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+        <img src="./images/ic_skills.png" alt="" srcset="">
+      </div>
+    </section>
+  </div>
+
   <div class="get-started-background">
     <img src="./images/image_geometry_3.png" alt="" class="img-before-get-started">
+    <img src="./images/image-geometry_6.png" alt="" class="img-before-get-started_desktop" srcset="">
+    <img src="./images/image-geometry_7.png" class="image-after-started-image" alt="" srcset="">
     <section class="get-started-section">
       <h1 class="section-title">Get Started</h1>
       <div class="greetings">
-        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</h3>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</h3>
       </div>
 
       <!-- form start -->
```

Comment:
As the previous reviewer requested, please follow the Figma template and
- Please make sure label are aligned with the input areas. And text area has the total width of the name and email areas.
<img width="1440" alt="Screenshot 2023-02-25 at 8 23 06 PM" src="https://user-images.githubusercontent.com/25887253/221371148-4ce3183b-7bd1-4569-94be-a1408e6512e3.png">
<img width="593" alt="Screenshot 2023-02-25 at 8 34 14 PM" src="https://user-images.githubusercontent.com/25887253/221371239-f78b0a16-b213-47bd-966a-0a1e2d01a08d.png">

Reply:
fixes